scripts/rabbitmq-server

```diff
@@ -151,6 +154,24 @@ stop_rabbitmq_server() {
     fi
 }
 
+check_start_params() {
+    check_not_empty "RABBITMQ_BOOT_MODULE" $RABBITMQ_BOOT_MODULE
+    check_not_empty "RABBITMQ_NAME_TYPE" $RABBITMQ_NAME_TYPE
+    check_not_empty "RABBITMQ_NODENAME" $RABBITMQ_NODENAME
+    check_not_empty "SASL_BOOT_FILE" $SASL_BOOT_FILE
+    check_not_empty "RABBITMQ_IO_THREAD_POOL_SIZE" $RABBITMQ_IO_THREAD_POOL_SIZE
+}
+
+check_not_empty() {
```

Comment:
It's possible to avoid duplicating variable name at a call site with a function body like this:
```shell
local name="${1:?}"
local value
eval value=\$$name
if [ -z "$value" ]; then
# error 
....
```